vertx-service-discovery/src/main/java/examples/HTTPEndpointExamples.java

```diff
@@ -73,6 +74,30 @@ public void example2(ServiceDiscovery discovery) {
     });
   }
 
+  public void example2_webclient(ServiceDiscovery discovery) {
+    // Get the record
+    discovery.getRecord(new JsonObject().put("name", "some-http-service"), ar -> {
+      if (ar.succeeded() && ar.result() != null) {
```

Comment:
Because when using service references, getting no service reference is not a failure. It just means there are no services at the moment.